util/ci_utils.sh

```diff
@@ -140,7 +146,7 @@ build_pip_package() {
     echo "Building Open3D wheel"
     options="$(echo "$@" | tr ' ' '|')"
 
-    BUILD_FILAMENT_FROM_SOURCE=OFF
+    BUILD_FILAMENT_FROM_SOURCE=ON
```

Comment:
This only needs to be built once (or when the filament version changes), so we should keep `BUILD_FILAMENT_FROM_SOURCE=OFF`. Please upload the filament binaries for ARM macOS (drag and drop as response to this comment) (as built by cmake during Open3D build), and I'll upload it to the download site with other filament binaries.

Reply:
Here are the arm64 binaries [filament-v1.9.19-macos_arm64.tgz](https://github.com/isl-org/Open3D/files/14736882/filament-v1.9.19-macos_arm64.tgz) from [this CI build](https://github.com/isl-org/Open3D/actions/runs/8408058252). Please upload them to https://github.com/isl-org/open3d_downloads/releases/filament/, I'll update the link.